lca-cli/postpivot/postpivot.go

```diff
@@ -541,3 +554,47 @@ func (p *PostPivot) applyNMStateConfiguration(seedReconfiguration *clusterconfig
 
 	return nil
 }
+
+// createPullSecretFile creates auth file on filesystem in order to be able to pull images
+// and runs createPullSecretManifest to write secret in manifests folder
+func (p *PostPivot) createPullSecretFile(pullSecret, pullSecretFile, pullSecretManifest string) error {
+	// TODO: Should return error in the future as cluster will not be operational without it
+	if pullSecret == "" {
+		p.log.Infof("Pull secret was not provided")
+		return nil
+	}
+	p.log.Infof("Move seed PS file aside")
+	if err := utils.MoveFileIfExists(pullSecretFile, pullSecretFile+"_seed"); err != nil {
```

Comment:
I don't think the seed PS should be there, see https://github.com/openshift-kni/lifecycle-agent/pull/256

Reply:
If it will not be there, nothing will happened, though i assume it should be just empty